scripts/ci-install-mdbook.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Copyright 2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+VERSION=${1}
+OUTPUT_PATH=${2}
+
+# Ensure the output folder exists
+mkdir -p "${OUTPUT_PATH}"
+
+# Get what release to download
+RELEASE_NAME=""
+case "$OSTYPE" in
+  darwin*) RELEASE_NAME="x86_64-apple-darwin.tar.gz"  ;;
+  linux*)  RELEASE_NAME="x86_64-unknown-linux-gnu.tar.gz" ;;
+#  msys*)    echo "WINDOWS" ;;
+  *)        echo "No mdBook release available for: $OSTYPE" && exit 1;;
+esac
+
+# Download and extract the mdBook release
+curl -L "https://github.com/rust-lang/mdBook/releases/download/${VERSION}/mdbook-${VERSION}-${RELEASE_NAME}" | tar -xvz -C "${OUTPUT_PATH}"
```

Comment:
We should verify if this also works on arm osx 🤔

Reply:
It works through Rosetta magic 🙂. There's a native build on brew, but only x86 on their release page.

Reply:
Sounds good 👍